.gitignore

```diff
@@ -1,4 +1,7 @@
 .vscode
 .DS_Store
 .env
+.api
+.database
 node_modules
+e2e-tests/cypress/*
```

Comment:
will this not ignore e2e tests?

Reply:
Good spot, have added exceptional directories.